### PR TITLE
[Enhancement] Rename cloud native aws config

### PR DESCRIPTION
**Pull request by xiangguangyxg**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Problem Summary(Required) ：

Rename cloud native aws config

## Checklist:

- [x] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [x] 3.0
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/common/Config.java [hunk 1]
```diff
@@ -1822,35 +1822,32 @@ public class Config extends ConfigBase {
...
-    /**
-     * AWS credential type, can be one of the following value, case-sensitive
-     * - "simple"
-     * - "instance_profile"
-     * - "assume_role"
-     */
     @ConfField
-    public static String cloud_native_aws_credential_type = "simple";
-    // Configurations for aws credential_type = "simple"
+    public static String aws_s3_use_instance_profile = "false";
```
Comment (kevincai) [inline fe/fe-core/src/main/java/com/starrocks/common/Config.java:1833]: make it boolean?

File: fe/fe-core/src/main/java/com/starrocks/common/Config.java [hunk 2]
```diff
@@ -1822,35 +1822,32 @@ public class Config extends ConfigBase {
...
-    public static String cloud_native_assume_role_credential_arn = "";
+    public static String aws_s3_external_id = "";
+
+    /**
+     * Enables or disables SSL connections to AWS services.
+     * Not used for now
+     */
     @ConfField
-    public static String cloud_native_assume_role_credential_external_id = "";
+    public static String aws_s3_enable_ssl = "true";
```
Comment (kevincai) [inline fe/fe-core/src/main/java/com/starrocks/common/Config.java:1850]: can remove it for now, will add it back when support this option.

File: fe/fe-core/src/main/java/com/starrocks/staros/StarMgrServer.java [hunk 3]
```diff
@@ -261,4 +262,26 @@ public long getMaxJournalId() {
     public long getReplayId() {
         return getJournalSystem().getReplayId();
     }
+
+    public static String[] getBucketAndPrefix() {
+        int index = Config.aws_s3_path.indexOf('/');
```
Comment (kevincai) [inline fe/fe-core/src/main/java/com/starrocks/staros/StarMgrServer.java:267]: can consider use `Config.aws_s3_path.split("/", 2)`

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 29 / 29 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/common/Config.java|9|9|100.00%|[]|
|:large_blue_circle:|com/starrocks/staros/StarMgrServer.java|20|20|100.00%|[]|

File: fe/fe-core/src/main/java/com/starrocks/common/Config.java (hunk 1 above)
Comment (xiangguangyxg) [reply fe/fe-core/src/main/java/com/starrocks/common/Config.java]: done

File: fe/fe-core/src/main/java/com/starrocks/common/Config.java (hunk 2 above)
Comment (xiangguangyxg) [reply fe/fe-core/src/main/java/com/starrocks/common/Config.java]: done

File: fe/fe-core/src/main/java/com/starrocks/staros/StarMgrServer.java (hunk 3 above)
Comment (xiangguangyxg) [reply fe/fe-core/src/main/java/com/starrocks/staros/StarMgrServer.java:272]: hard to use ...

File: fe/fe-core/src/main/java/com/starrocks/staros/StarMgrServer.java [hunk 4]
```diff
@@ -126,16 +126,22 @@ public void initialize(BDBEnvironment environment, String baseImageDir) throws I
         // HDFS related configuration
         com.staros.util.Config.HDFS_URL = Config.cloud_native_hdfs_url;
         // AWS related configuration
-        com.staros.util.Config.S3_BUCKET = Config.cloud_native_aws_s3_bucket;
-        com.staros.util.Config.S3_REGION = Config.cloud_native_aws_s3_region;
-        com.staros.util.Config.S3_ENDPOINT = Config.cloud_native_aws_s3_endpoint;
-        com.staros.util.Config.S3_PATH_PREFIX = Config.cloud_native_aws_s3_path;
+        String[] bucketAndPrefix = getBucketAndPrefix();
```
[the remark this answers is not in the document]
Comment (xiangguangyxg) [reply fe/fe-core/src/main/java/com/starrocks/staros/StarMgrServer.java:133]: done

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 39 / 45 (86.67%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/staros/StarMgrServer.java|30|36|83.33%|[129, 130, 139, 140, 145, 146]|
|:large_blue_circle:|com/starrocks/common/Config.java|9|9|100.00%|[]|

**Review by wyb — APPROVED** [review]

**Comment by xiangguangyxg** [comment]
run starrocks_fe_unittest

**Review by sduzh — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 40 / 48 (83.33%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/staros/StarMgrServer.java|31|39|79.49%|[127, 128, 133, 134, 143, 144, 149, 150]|
|:large_blue_circle:|com/starrocks/common/Config.java|9|9|100.00%|[]|

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Pull request merged**